### PR TITLE
[7735] override wagtail-metadata template to exclude meta title

### LINKED ISSUE #7735: Pages have two `<title>` elements
view-source:https://foundation.mozilla.org/en/ shows that we have two `<title>` elements, one up top, and one halfway down the `<head>` element.

### PR DESCRIPTION
**Pull request by richbrennan**
Closes #7735

The `wagtail-metadata` package is conflicting with manually set HTML elements, so this PR overrides the `wagtail-metadata` template to remove the meta title, which is already being set in `base.html`

**Link to sample test page**:  view-source:https://foundation-s-fix-7735-m-dkbd7x.herokuapp.com/en/

### CODE REVIEW

**Review by Pomax — CHANGES_REQUESTED** [review]
tiny change + a question

File: network-api/networkapi/templates/wagtailmetadata/parts/tags.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% extends "wagtailmetadata/parts/tags.html" %}
+
+{% block meta %}
+<meta itemprop="url" content="{{ object.get_meta_url }}"/>
+<meta itemprop="name" content="{{ object.get_meta_title }}">
+<meta itemprop="description" content="{{ object.get_meta_description }}" />
+{% if meta_image %}<meta itemprop="image" content="{{meta_image}}" />{% endif %}
+
```
[the remark this answers is not in the document]
Comment (richbrennan) [reply network-api/networkapi/templates/wagtailmetadata/parts/tags.html]: Done!

File: .gitignore [hunk 2]
```diff
@@ -23,12 +23,13 @@ eggs/
 .eggs/
 lib/
 lib64/
-parts/
+parts/*
```
[the remark this answers is not in the document]
Comment (richbrennan) [reply .gitignore:26]: No, without the `*` it doesn't even read `parts/` in order to be able to add `tags.html` back in https://stackoverflow.com/a/2820310

[the remark this answers is not in the document]
Comment (richbrennan) [reply .gitignore:26]: No, without the `*` it doesn't read the contents of `parts/` to be able to add `tags.html` back in https://stackoverflow.com/a/2820310

**Review by Pomax — APPROVED** [review]

**Pull request merged**